includes/classes/bmz_image_handler.class.php

```diff
@@ -927,25 +928,24 @@ protected function calculate_gravity($canvaswidth, $canvasheight, $overlaywidth,
      *
      * @return false|GdImage|mixed|resource
      */
-    protected function load_imageGD($src_name)
+    protected function load_imageGD($src_name,$retry = false)
```

Comment:
Where's the usage of this newly-added function parameter?  Either use it (and add a space after the `,`) or remove it.